src/planwise/client/projects2/components/settings.cljs

```diff
@@ -15,7 +15,9 @@
             [planwise.client.ui.common :as ui]
             [planwise.client.ui.filter-select :as filter-select]
             [planwise.client.ui.rmwc :as m]
-            [planwise.client.utils :as utils]))
+            [planwise.client.utils :as utils]
+            [planwise.model.starting-project :as project-start-model]
```

Comment:
As long as the specs are loaded, you don't actually need this require here. The spec registry is global, so you can refer the spec later with the fully qualified namespace.